src/api/app/models/package.rb

```diff
@@ -933,9 +933,12 @@ def jobhistory_list(project, repository, arch, options = {})
     options[:limit] = 100 if options[:limit].blank?
     options[:package] = name if options[:package].blank?
 
-    results = Jobhistory.find_hashed(project: project.name, package: options[:package],
-                                     repository: repository, arch: arch,
-                                     limit: options[:limit])
+    begin
+      results = Backend::Api::BuildResults::JobHistory.all_for_package(project.name, options[:package], repository, arch, options[:limit])
```

Comment:
what about other variable name? same same for different things :see_no_evil:  `results_to_parse` or something like that :thinking:

Reply:
really? the code is IMO very clear as it's reusing the variable in a very close context

and not reusing the variable keeps one more variable's content in memory for the rest of the action. So I disagree with your suggestion.

Reply:
I collapsed this now (as done everywhere else). This gives super long lines - but if you prefer that over storing interim results, be it